### ISSUE TITLE
Incompatible Required Packages

### ISSUE THREAD

**Issue by kolojoe** [Paebbels/pyVHDLParser#35]
I am interested in using this tool and tried to use the VHDLParser debug tool, but it turns out that there are some incompatibilities with the packages that are currently released vs what the package says is acceptable.

The following requirements are causing issues.

```
pyTooling>=1.7.0
pyTooling.TerminalUI>=1.5.5
```
The latest version of `pyTooling` is `2.0.1` which no longer includes the `Singleton` metaclass and caused this line to fail:
```
  File "...\pyVHDLParser\pyVHDLParser\CLI\VHDLParser.py", line 36, in <module>
    from pyTooling.MetaClasses            import Singleton
ImportError: cannot import name 'Singleton' from 'pyTooling.MetaClasses' (...Python\Python39\lib\site-packages\pyTooling
\MetaClasses\__init__.py)
```

After falling back to `pyTooling==1.7.0` there were further issues, but those were indirect import issues not necessarily caused by `pyVHDLParser`.

I upgraded `pyTooling==1.9.5` but that was incompatible with `pyTooling.TerminalUI==1.5.8`. After downgrading back to `pyTooling.TerminalUI==1.5.5` it finally worked.

# Suggestion
Require `pyTooling` to be between `1.7.0` and `1.9.5`. I haven't tested later version before `2.0.1` so perhaps just version <= 2 would work.
Require `pyTooling.TerminalUI==1.5.5`.

**Comment by Paebbels** [comment]
Thanks for reporting this issue.
I'll fix this in the following days.

pyTooling and pyTooling.TerminalUI, which are also written by me, got a major and braking update. I'm now going to update all packages (+20) using these dependencies. I'll need some time to fix all of them.

**Comment by Paebbels** [comment]
This will be addressed in #48.

**Comment by Paebbels** [comment]
Unit tests on CI are passing.
